motion_config.html

```diff
@@ -2025,17 +2174,19 @@ <h3><a name="setup_mode"></a> setup_mode </h3>
         labeled area (assuming despeckle is enabled and with the 'l' at the end) is blue. It is
         only the blue areas which is counted as Motion. If smartmask is enabled you see this as red areas.
 
-        Here is a suggestion how to initially setup Motion.
+        Here is a suggestion how to initially setup Motion.  (There are other ways to accomplish this.)
         <p></p>
         <ul>
           <li> Disable despeckle (comment it out in motion.conf).</li>
           <li> Disable smartmask</li>
-          <li> Enable both http control and webcam by setting port numbers. Example 8080 for control and 8081 for webcam.</li>
+          <li> Enable both the webcontrol and streams make sure to specify the webcontrol_parms. The below
+            assumes that the webcontrol is specified as port 8080 and the stream as 8081
+          </li>
           <li> Start Motion in setup mode</li>
-          <li> View the webcam stream. Either with Iceweasel or with Firefox by entering in the address of
+          <li> View the webcam stream via a browser by entering in the address of
           <a href="http://localhost:8081/" rel="nofollow" target="_top">http://localhost:8080/</a>
```

Comment:
I found a typo into line 2187. The wrong port number 8080 is used into link text. The right one is http://localhost:8081/